todo_app/app.py

```diff
@@ -1,11 +1,19 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
+@app.route('/items')
 @app.route('/')
```

Comment:
Is there a reason you've chosen to give this a second route? No particular reason it's an issue, but all else being equal having only one way to do things means less space for something to go wrong!

Reply:
I was testing it out having multiple routes. Comments are noted and I have updated the project so there is a singular route.